components/link/link.js

```diff
@@ -173,11 +174,12 @@ class Link extends LocalizeCoreElement(FocusMixin(LitElement)) {
 		};
 		const spanClasses = {
 			'truncate': this.lines > 1,
-			'truncate-one': this.lines === 1
+			'truncate-one': this.lines === 1,
+			'vdiff-target': true
```

Comment:
Necessary because `display: contents` makes the component itself essentially invisible.